crossandeancommands/make_nexus.py

```diff
@@ -0,0 +1,81 @@
+"""
+Create a nexus file from the most recent data on Edictor.
+"""
+from lingpy import Wordlist
+from lingpy.convert.strings import write_nexus
+from pathlib import Path
+
+
+def run(args):
+
+    wl = Wordlist.from_cldf(
+            Path("./", "cldf", "cldf-metadata.json").as_posix(),
+            # columns to be loaded from CLDF set
+            columns=(
+                "language_id",
+                "parameter_id",
+                "segments",
+                "language_subgroup",
+                "borrowing",
+                "cogid_cognateset_id"
+                ),
+            # a list of tuples of source and target
+            namespace=(
+                ("language_id", "doculect"),
+                ("parameter_id", "concept"),
+                ("segments", "tokens"),
+                ("language_subgroup", "subgroup"),
+                ("cogid_cognateset_id", "cogids")
+                )
+            )
+
+    # These are the blacklists for the nexus file
+    blacklist_languages = [
+            'CuzquenoAntiguo',
+            'Anonimo',
+            'SantoTomas',
+            ]
+
+    blacklist_concepts = [
+            'star',
+            'green',
+            'with',
+            'small',
+            'lip',
+            'wing',
+            'breast',
+            'suck,to',
+            'husband',
+            'branch'
+            ]
+
+    blacklist_subgroups = [
+            'Aymara',
+            'Uru-Chipaya'
+            ]
+
+    blacklist_borrowings = [
```

Comment:
checking for membership is faster for sets, by the way, so you may want to use a set instead of a list here.